src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -291,6 +291,24 @@ pub struct NamingAuthority<'a> {
     pub text: Option<DisplayText<'a>>,
 }
 
+type SequenceOfDisplayTexts<'a> = common::Asn1ReadableOrWritable<
+    asn1::SequenceOf<'a, DisplayText<'a>>,
+    asn1::SequenceOfWriter<'a, DisplayText<'a>, Vec<DisplayText<'a>>>,
+>;
+
+type SequenceOfObjectIdentifiers<'a> = common::Asn1ReadableOrWritable<
+    asn1::SequenceOf<'a, asn1::ObjectIdentifier>,
+    asn1::SequenceOfWriter<'a, asn1::ObjectIdentifier, Vec<asn1::ObjectIdentifier>>,
+>;
+
+pub struct ProfessionInfo<'a> {
+    pub naming_authority: Option<NamingAuthority<'a>>,
```

Comment:
Please add a commented out `#[explicit(0)]` since that will be needed when we add the derives.

Reply:
@alex done, please verify. Do you think it makes sense to also add the `#[derive(asn1::Asn1Read, asn1::Asn1Write)]` headers as well, commented out for now?

Reply:
sure

Reply:
@alex done!